### PR TITLE
[DeckGL] Raise error with null values

### PR DESCRIPTION
**Pull request by hughhhh**
Really vague message with null values. Want to be more transparent about the errors in the underlying code for showing data in deckgl

@mistercrunch

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -2107,6 +2108,10 @@ def tupleify(s):
             df[key] = list(zip(latlong.apply(lambda x: x[0]),
                                latlong.apply(lambda x: x[1])))
             del df[spatial.get('geohashCol')]
+
+        if df[key] is None:
```
Comment (mistercrunch) [inline superset/viz.py]: (optional)`df.get(key) is None:` would protect against KeyError (not sure if they're possible in this context)

File: superset/viz.py [hunk 2]
```diff
@@ -2107,6 +2108,10 @@ def tupleify(s):
...
+            raise NullValueException('Some rows in this query contain NULL values!')
```
Comment (mistercrunch) [inline superset/viz.py]: Msg could be a bit more precise and use i18n wrapper: `_(Encountered invalid NULL spatial entry, please consider filtering those out)`

Comment (mistercrunch) [reply superset/viz.py]: @vylc wording ok?

Comment (hughhhh) [reply superset/viz.py]: Is there a python lib for ` i18n`? i thought is was only available for JS, i was searching the code for some example and couldn't find any.

Comment (mistercrunch) [reply superset/viz.py]: in this module it's available from the line `from flask_babel import lazy_gettext as _`

Comment (mistercrunch) [reply superset/viz.py]: the setup is kind of sweet where we use the same framework for both the backend and frontend.

**Comment by mistercrunch** [comment]
LGTM

**Pull request merged**